### PR TITLE
Completed big-o-notation exercise. - Barbara Liao

### PR DESCRIPTION
**Pull request by barbara-liao**
image

### CODE REVIEW

File: big-o-notation/big-o.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(n)
+    i++                             // 2 * n = O(n)
+  ) {
+    const word = words[i];          // 2 * n = O(n)
+    let isUnique = true;            // 1 * n = O(n)
+    for (
+      let c = 0;                    // 1 * n^2 = O(n^2)
```
Comment (thebearingedge) [inline big-o-notation/big-o.js:30]: This initialization expression only runs one time per word, so this line would be linear 1 * n = O(n)

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: big-o-notation/big-o.js (hunk 1 above)
Comment (barbara-liao) [reply big-o-notation/big-o.js]: ohh ok!

File: big-o-notation/big-o.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ function uniqueQuadratic(words) {
     const word = words[i];          // 2 * n = O(n)
     let isUnique = true;            // 1 * n = O(n)
     for (
-      let c = 0;                    // 1 * n^2 = O(n^2)
+      let c = 0;                    // 1 * n^2 = O(n)
```
Comment (thebearingedge) [inline big-o-notation/big-o.js:30]: Right, but you would change your `n^2` to just `n`

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: big-o-notation/big-o.js (hunk 2 above)
Comment (barbara-liao) [reply big-o-notation/big-o.js]: whoops, missed that first n^2!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**